src/v2/registry.py

```diff
@@ -1,7 +1,38 @@
+from gi.repository import GObject
+
 from hack_sound_server.registry import Registry as _Registry
 
 
 class Registry(_Registry):
+    __gsignals__ = {
+        "player-added": (GObject.SignalFlags.RUN_FIRST, None, (object, )),
+        "player-removed": (GObject.SignalFlags.RUN_FIRST, None, ())
+    }
+
     def __init__(self):
         super().__init__()
         self.players_by_bus_name = {}
+
+    def get_player(self, app_id):
+        """
+        Obtains a player given its app id (a well-known name).
+
+        Arguments:
+            app_id (str): A well known name, ie: "com.endlessm.Fizzics".
+
+        Returns:
+            A `Player` object if exists in the registry. Otherwise, `None`.
+        """
+        return self.players_by_bus_name.get(app_id)
+
+    def add_player(self, player):
```

Comment:
I would suggest merging the v2 Registry into the base class Registry. It seems like these extra bits will not do any harm if used in v1.